command/tokenize_test.go

```diff
@@ -56,3 +62,13 @@ func TestTokenizeErrors(t *testing.T) {
 		assert.IsType(t, TokenizeError{}, err, in)
 	}
 }
+
+func TestIsQuotationMark(t *testing.T) {
```

Comment:
I've found that it makes debugging easier if we stick to a naming convention for tests (which is, granted, incompletely applied) of `Test${package}${tested function}`. So this would be `TestCommandQuotationMarkCategory`.

Reply:
Isn't that a little redundant, since the tests are sorted by package when run already? Definitely changing the test name anyway, I renamed the function after writing the test and totally forgot about it.